sys-apps/baselayout/baselayout-9999.ebuild

```diff
@@ -113,6 +113,17 @@ src_install() {
 	for sym in "${!LIB_SYMS[@]}" ; do
 		dosym "${LIB_SYMS[$sym]}" "${sym}"
 	done
+
+	if use cros_host; then
+		# We assume that SDK already has the user and group database.
```

Comment:
The folder `/home/core/` is gone from the SDK which could be related to this change. `cork` fails to use the SDK because `useradd: cannot create directory /home/build` but I don't know if this is also related.

Reply:
Ah, no, this is just moved from below, then my search continues.

Reply:
This was the problem because of the changed behavior. I wonder if we could use UIDs for systemd-tmpfiles. Anyway, the PR with a workaround is here https://github.com/flatcar-linux/coreos-overlay/pull/639